shared/src/main/scala/squants/motion/AngularAcceleration.scala

```diff
@@ -58,15 +67,15 @@ trait AngularAccelerationUnit extends UnitOfMeasure[AngularAcceleration] with
 }
 
 object RadiansPerSecondSquared extends AngularAccelerationUnit with PrimaryUnit with SiUnit{
-  override val symbol: String = "rad/s²"
+  val symbol = Radians.symbol + "/s²"
 }
 
 object DegreesPerSecondSquared extends AngularAccelerationUnit {
-  val symbol = "°/s²"
+  val symbol = Degrees.symbol + "/s²"
   val conversionFactor = Degrees.conversionFactor
 }
 
-object GradiansPerSecondSquared extends AngularAccelerationUnit {
+object GradsPerSecondSquared extends AngularAccelerationUnit {
```

Comment:
I made it `Grads` instead of `Gradians` because that was the convention used in `AngularVelocity`. However, I agree with you about `Gradians`  being clearer, and I will change it. Do you also want to change that in `AngularVelocity`?